src/components/Autocomplete/autocomplete.scss

```diff
@@ -0,0 +1,63 @@
+@import 'globals';
+
+$default-border-radius: map-get($form-fields-defaults, 'border-radius');
+
+.input {
+  @include map-declarations($form-fields-defaults, $form-fields-custom);
+  display: block;
+  width: 50%;
+  transition: border-radius 150ms;
+}
+
+.open .input {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.valid {
+  border-color: color('grey');
+}
+
+.valid:focus {
+  border-color: color('dimmed');
+}
+
+.invalid {
+  border-color: color('error');
+}
+
+.list {
+  @include map-declarations($form-fields-defaults, $form-fields-custom);
+  padding: 0;
+  border-top: 0;
+  border-color: color('grey');
+  border-radius: 0 0 $default-border-radius $default-border-radius;
+  width: 50%;
+  list-style: none;
+}
+
+.suggestion {
+  @include map-declarations($form-fields-defaults, $form-fields-custom);
+  border: 1px solid color('grey-lite');
+  border-left: 0;
+  border-right: 0;
```

Comment:
i would just set explicit `border-top` and `border-bottom` - rather than undoing